docs/project/changelog.md

```diff
@@ -32,8 +32,14 @@ substitutions:
   {pr}`1849`
 
 - {{Fix}} Webpack building compatibility issues and a {any}`loadPyodide <globalThis.loadPyodide>`
-  runtime issue due to webpack are solved. 
-  {pr}`1900`  
+  runtime issue due to webpack are solved.
+  {pr}`1900`
+
+- {{Enhancement}} Added a {any}`pyodide.pyimport` api to import a Python module
+  and return it as a `PyProxy`. Note that this does a different thing than the
+  original `pyimport` api: it imports a package and returns it without adding
+  the package to the global scope.
```

Comment:
But  it would still be in the sys.modules, if you access it from Python right? I'm just curious, no need to add it to the changelog.

Reply:
Yes